webapp/src/js/services/enketo.js

```diff
@@ -306,6 +306,13 @@ angular.module('inboxServices').service('Enketo',
         .on('click.pagemode', function() {
           form.pages.next()
             .then(function(newPageIndex) {
+              if(!newPageIndex){
+                if ($wrapper.find('.invalid-required').length > 0) {
+                  $($wrapper).find('.next-page').attr('disabled','disabled');
+                } else {
+                  $($wrapper).find('.next-page').removeAttr('disabled');
```

Comment:
I think the $wrapper is already a jquery object  (it's used on line 310) so wrapping it in `$()` here and on line 311 does nothing.